firestore/integration_test_internal/src/settings_test.cc

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 Google LLC
+
+#include <future>
+#include <map>
+#include <string>
+
+#include "firebase/firestore.h"
+#include "firestore_integration_test.h"
+
+namespace firebase {
+namespace firestore {
+
+using SettingsTest = FirestoreIntegrationTest;
+
+TEST_F(SettingsTest, Equality) {
```

Comment:
Note: this test doesn't seem to use any functionality from `FirestoreIntegrationTest`. I think it can be just `TEST(SettingsTest, Equality)` (and you can remove the include).

Reply:
Done